CHANGES.md

```diff
@@ -1,5 +1,36 @@
 # Changes
 
+## edge-23.3.2
+
+This edge release continues to improve dynamic Policy statuses and
+introduces support for header-based routing.
+
+* Destination Controller
+  * Added OutboundPolices API, for use by `linkerd-proxy` to route
+    outbound traffic
+  * Improved diagnostic log messages
+  * Fixed sending of spurious profile updates
+
+* Proxy
+  * Utilize new OutboundPolices API, supporting Gateway API-style routes
+    in the outbound proxy
+  * Proxies in ingress and gateway configurations may now
+    discover policies by name instead of network address
+
+* Policy Controller
+  * Utilize `policy-controller-write` Lease to avoid multiple writers
```

Comment:
Do we want to phrase this in terms of being able to run multiple replicas of the policy controller / run it in HA mode?

Reply:
Good idea, I'm reframing